flatten-array.json

```diff
@@ -0,0 +1,19 @@
+{
+   "cases": [
+      {
+         "description": "flattens array with null and integers present",
+         "input": [0, 2, [[2, 3], 8, 100, null, [[null]]], -2],
+         "expected": [0, 2, 2, 3, 8, 100, -2]
+      },
+      {
+         "description": "flattens array with just integers present",
+         "input": [1,[2,3,4,5,6,7],8],
+         "expected": [1,2,3,4,5,6,7,8]
```

Comment:
The smallest example of this would be `[1, [2], 3]` or, if you're worried about having a nested array of one element, maybe `[1, [2, 3], 4]`.